workbench-core/infrastructure/docs/WorkbenchDynamodb.md

```diff
@@ -0,0 +1,48 @@
+# WorkbenchDynamodb CDK Construct
+
+## Description
+**A CDK Construct with some default values and is intended solely for use in our integration tests and is not intended for advanced applications such as creating GlobalTables with replicationRegion**:
+1. billingMode: Either provide your billingMode or it sets the default to BillingMode.PAY_PER_REQUEST
+1. encryption: TableEncryption.CUSTOMER_MANAGED // always
+1. encryptionKey: Either provide your own encryption key or it creates an encryptionKey with rotation enabled
+1. pointInTimeRecovery: true // always
+
+## Usage
+
+### Installing the package
+
+Using NPM:
+```bash
+npm install @aws/workbench-core-infrastructure
```

Comment:
is this is "solely for use in our integration tests", are we going to publish this via NPM?

Reply:
@DaveMadsen I have made it more generic. All the CDK constructs should work well for SWB and MAF both !